FAULT TYPE: OPENSTACK_THROW_EXCEPTION-NETWORK
TARGET COMPONENT: /nova/network/neutronv2/api.py
TARGET CLASS: API
TARGET FUNCTION DEF: _get_available_networks+363;8
FAULT POINT: _ensure_requested_network_ordering((lambda x: x['id']), nets, net_ids)
